js/ui/overlay/ui.overlay.js

```diff
@@ -1200,10 +1211,18 @@ const Overlay = Widget.inherit({
         this._animateDeferred = animateDeferred;
         this.option('visible', showing);
 
-        animateDeferred.promise().done(() => {
-            delete this._animateDeferred;
-            result.resolveWith(this, [this.option('visible')]);
-        });
+        animateDeferred.promise()
+            .done(
+                () => {
```

Comment:
why did u decide to start a function from a new line?

Reply:
I am an artist, i see it in this way!